### ISSUE TITLE
A Python tool for lv_font_conv

### ISSUE THREAD

**Issue by Linjieqiang** [lvgl/lv_font_conv#115]
Is there anyone in the community who can make a font conversion tool using python language, which will be very convenient to apply in CI?

**Comment by kisvegabor** [comment]
I agree that it'd be great, however I don't know anyone working on it. 🙁

**Comment by Christer-Ekholm** [comment]
I used instructions from here https://anga.blue/blog/windows-executable-file-exe-node-js
to compile lv_font_conv to be executable.  Then it can be used in our CI.

**Comment by kisvegabor** [comment]
If are about having an executable I'd go for `pkg` as it support Window, Linux and Mac executable too.
